jwql/website/apps/jwql/views.py

```diff
@@ -614,7 +614,10 @@ def jwqldb_table_viewer(request):
     try:
         tablename = request.POST['db_table_select']
     except KeyError:
-        tablename = None
+        if tablename_param:
+            tablename = tablename_param
```

Comment:
if you are sending a default of `None` then you dont need the if/else.  You can simply say:
```
except KeyError:
     tablename = tablename_param
```

Reply:
I like this idea, if `tablename_param` is None, we still achieve the same goal here.

Reply:
@mfixstsci , are you making that change or should I merge as is?